src/CaptureFileInfo/LoadCaptureWidgetTest.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+#include <qapplication.h>
+
+#include <QApplication>
+#include <QPushButton>
+#include <QRadioButton>
+#include <QTableView>
+#include <QTest>
+#include <filesystem>
+
+#include "CaptureFileInfo/LoadCaptureWidget.h"
+#include "CaptureFileInfo/Manager.h"
+#include "OrbitBase/ExecutablePath.h"
+
+namespace orbit_capture_file_info {
+
+constexpr const char* kOrgName = "The Orbit Authors";
+
+TEST(LoadCaptureWidget, IsActiveSetActive) {
+  QCoreApplication::setOrganizationName(kOrgName);
+  QCoreApplication::setApplicationName("LoadCaptureWidget.IsActiveSetActive");
+
+  LoadCaptureWidget widget{};
+
+  auto* radio_button = widget.findChild<QRadioButton*>("radioButton");
+  ASSERT_TRUE(radio_button != nullptr);
+  auto* select_file_button = widget.findChild<QPushButton*>("selectFileButton");
+  ASSERT_TRUE(select_file_button != nullptr);
+  auto* table_view = widget.findChild<QTableView*>("tableView");
+  ASSERT_TRUE(table_view != nullptr);
+
+  // The widget needs to be show so the detaching of the radio button takes place. Otherwise the
+  // radioButton is not detached from the contentFrame and will be disabled when the widget is set
+  // inactive.
+  widget.show();
+  QApplication::processEvents();
+
+  // default is active
+  EXPECT_TRUE(widget.IsActive());
+  EXPECT_TRUE(select_file_button->isEnabled());
+  EXPECT_TRUE(table_view->isEnabled());
+  // radio button is always enabled
+  EXPECT_TRUE(radio_button->isEnabled());
+
+  // set active false
+  widget.SetActive(false);
+  EXPECT_FALSE(widget.IsActive());
+  EXPECT_FALSE(select_file_button->isEnabled());
+  EXPECT_FALSE(table_view->isEnabled());
+  EXPECT_FALSE(radio_button->isChecked());
+  // radio button is always enabled
+  EXPECT_TRUE(radio_button->isEnabled());
+
+  // set active true
+  widget.SetActive(true);
+  EXPECT_TRUE(widget.IsActive());
+  EXPECT_TRUE(select_file_button->isEnabled());
+  EXPECT_TRUE(table_view->isEnabled());
+  EXPECT_TRUE(radio_button->isChecked());
+  // radio button is always enabled
+  EXPECT_TRUE(radio_button->isEnabled());
+
+  // set active to false and then activate by clicking on the radiobutton
+  widget.SetActive(false);
+  bool signal_received = false;
+  QObject::connect(&widget, &LoadCaptureWidget::Activated,
+                   [&signal_received]() { signal_received = true; });
+  QTest::mouseClick(radio_button, Qt::MouseButton::LeftButton, Qt::KeyboardModifier::NoModifier);
+  EXPECT_TRUE(signal_received);
```

Comment:
No need to change this, but I wanted to mention that there is QSignalSpy in QTest (https://doc.qt.io/qt-5/qsignalspy.html) to check whether a signal was emitted or not. Maybe this comes handy in the future.

Reply:
Thanks!